### ISSUE TITLE
Gurobi Computational problem!

### ISSUE THREAD

**Issue by WenJieWan** [eth-sri/eran#5]
Hi!
i want to use my cifar model to verify the network, but i have met a problem when l use Eran.
my model file type is h5 ,so i use this convert method (https://github.com/Callmejp/DiplomaProject/tree/master/ChestNet) to extract weights by modifying the line 66 in tensorflow_translator.py to name.
I don't know whether it successfully convert model.
Although it can run in the eran.analyze_box function, it can't run in the verify_network_with_milp function. The gurobi optimizer doesn't has a feasible solution, because the status of model
is equal to GRB.Status.INFEASIBLE. It will cause a error "Unable to retrieve attribute objval".
So, can you tell me why my model can't compute by milp and how to modify it to compute successfully.
My network structure is shown as follows：

model = Sequential()
model.add(Convolution2D(nb_filters, kernel_size, padding='same', activation='relu', input_shape=input_shape))
model.add(Convolution2D(nb_filters, kernel_size, padding='same', activation='relu'))
model.add(MaxPooling2D())
model.add(Convolution2D(nb_filters2, kernel_size, padding='same', activation='relu'))
model.add(Convolution2D(nb_filters2, kernel_size, padding='same', activation='relu'))
model.add(MaxPooling2D())
model.add(Flatten())
model.add(Dense(64, activation='relu'))
model.add(Dense(nb_classes, activation='sigmoid'))
model.compile(optimizer='Adam',loss='categorical_crossentropy',metrics=['accuracy'])

Thanks for reading!

**Comment by GgnDpSngh** [comment]
Hi,

The complete verification with ERAN for networks with sigmoid activations is not supported at the moment, that is why you get an infeasible solution. We will add this feature in the future.

Cheers,

**Comment by GgnDpSngh** [comment]
Incomplete verification with DeepPoly or DeepZ should work on this network. Did you try running ERAN with these options?

Cheers,
